### ISSUE TITLE
Ember serve and eslint failure when plugin ember/recommended is enabled

### ISSUE THREAD

**Issue by eharrow** [ember-cli/eslint-plugin-ember#113]
Fab plugin BTW though I have an issue after upgrading to ember 2.14.0 and the ember/recommended plugin is enabled in my .eslintrc.js

running `ember serve` results in 

```The Broccoli Plugin: [SimpleConcatConcat] failed with:
TypeError: Cannot read property 'type' of null
    at Object.isMemberExpression (/myapp/node_modules/eslint-plugin-ember/lib/utils/utils.js:83:36)
    at EventEmitter.VariableDeclarator (/myapp/node_modules/eslint-plugin-ember/lib/rules/no-global-jquery.js:35:21)
    at emitOne (events.js:96:13)
    at EventEmitter.emit (events.js:188:7)
    at NodeEventGenerator.applySelector (/myapp/node_modules/eslint/lib/util/node-event-generator.js:265:26)
    at NodeEventGenerator.applySelectors (/myapp/node_modules/eslint/lib/util/node-event-generator.js:294:22)
    at NodeEventGenerator.enterNode (/myapp/node_modules/eslint/lib/util/node-event-generator.js:308:14)
    at CodePathAnalyzer.enterNode (/myapp/node_modules/eslint/lib/code-path-analysis/code-path-analyzer.js:602:23)
    at CommentEventGenerator.enterNode (/myapp/node_modules/eslint/lib/util/comment-event-generator.js:98:23)
    at Traverser.enter (/myapp/node_modules/eslint/lib/eslint.js:929:36)

The broccoli plugin was instantiated at: 
    at Concat.Plugin (/myapp/node_modules/broccoli-plugin/index.js:7:31)
    at new Concat (/myapp/node_modules/broccoli-concat/concat.js:38:10)
    at module.exports (/myapp/node_modules/broccoli-concat/index.js:26:10)
    at Function.EslintValidationFilter.create (/myapp/node_modules/broccoli-lint-eslint/lib/index.js:215:10)
    at Class.lintTree (/myapp/node_modules/ember-cli-eslint/index.js:44:19)
    at output.project.addons.map.addon (/myapp/node_modules/ember-cli/lib/broccoli/ember-app.js:649:22)
    at Array.map (native)
    at EmberApp.addonLintTree (/myapp/node_modules/ember-cli/lib/broccoli/ember-app.js:647:38)
    at EmberApp.lintTestTrees (/myapp/node_modules/ember-cli/lib/broccoli/ember-app.js:1191:26)
    at EmberApp.appTests (/myapp/node_modules/ember-cli/lib/broccoli/ember-app.js:1162:28)
```

.eslintrc.js:
```
module.exports = {
  root: true,
  parserOptions: {
    ecmaVersion: 2017,
    sourceType: 'module'
  },
  extends: [
    'eslint:recommended',
    'plugin:ember/recommended'
  ],
  env: {
    browser: true
  },
  rules: {
    // override rules' settings here
  }
};
```
Removing `plugin:ember/recommended` and it starts up ok.

package.json as follows:
```  "devDependencies": {
    "auto-changelog": "^0.3.1",
    "broccoli-asset-rev": "^2.4.5",
    "ember-ajax": "^3.0.0",
    "ember-c3": "^0.3.1",
    "ember-c3-shim": "^0.0.7",
    "ember-cli": "~2.14.0",
    "ember-cli-app-version": "^3.0.0",
    "ember-cli-babel": "^6.3.0",
    "ember-cli-bootstrap-datetimepicker": "^0.5.0",
    "ember-cli-bootstrap-sassy": "^0.5.6",
    "ember-cli-bootstrap3-popover": "^1.0.3",
    "ember-cli-clipboard": "^0.8.0",
    "ember-cli-dependency-checker": "^1.3.0",
    "ember-cli-eslint": "^3.0.0",
    "ember-cli-flash": "^1.4.3",
    "ember-cli-g-maps": "^0.5.14",
    "ember-cli-htmlbars": "^2.0.1",
    "ember-cli-htmlbars-inline-precompile": "^0.4.3",
    "ember-cli-inject-live-reload": "^1.4.1",
    "ember-cli-moment-shim": "3.3.0",
    "ember-cli-qunit": "^4.0.0",
    "ember-cli-sass": "^6.1.3",
    "ember-cli-shims": "^1.1.0",
    "ember-cli-sri": "^2.1.0",
    "ember-cli-string-helpers": "^1.4.0",
    "ember-cli-uglify": "^1.2.0",
    "ember-d3": "^0.2.0",
    "ember-data": "~2.14.3",
    "ember-data-model-fragments": "~2.14.0",
    "ember-export-application-global": "^2.0.0",
    "ember-font-awesome": "^3.0.5",
    "ember-i18n": "^5.0.2",
    "ember-load-initializers": "^1.0.0",
    "ember-radio-button": "^1.1.1",
    "ember-resolver": "^4.0.0",
    "ember-route-action-helper": "^2.0.5",
    "ember-scroll-to": "^0.6.4",
    "ember-select": "^0.3.0",
    "ember-source": "~2.14.0",
    "ember-string-helpers": "^1.0.1",
    "ember-truth-helpers": "^1.3.0",
    "ember-welcome-page": "^3.0.0",
    "eonasdan-bootstrap-datetimepicker-npm": "4.17.42",
    "eslint-plugin-ember": "^4.0.0",
    "git-changelog": "^1.1.2",
    "loader.js": "^4.2.3",
    "moment": "^2.18.1",
    "moment-timezone": "^0.5.13"
  },
```

**Comment by michalsnik** [comment]
Try `v4.1.2`

**Comment by eharrow** [comment]
Winner.  This resolves the issue.

**Comment by michalsnik** [comment]
The real issue remains in `no-global-jquery` rule though, but we'll work on it.

**Comment by michalsnik** [comment]
@eharrow We just released `v4.1.3` and you should be able to turn `no-global-jquery` rule on again:
```
// .eslintrc.js
module.exports = {
  ...
  rules: {
    ...
    'ember/no-global-jquery': 2
  }
}
```

It's not enabled by default now, but when you enable it please let me know whether the issue disappeared.

**Comment by eharrow** [comment]
Thanks that works as expected.
